ingestion/src/metadata/ingestion/source/dashboard/tableau/metadata.py

```diff
@@ -166,6 +167,7 @@ def prepare(self):
                 sheet_type=chart["sheetType"],
                 view_url_name=chart["viewUrlName"],
                 tags=[tag["label"] for tag in chart.get("tags", {}).get("tag") or []],
+                content_url=chart["contentUrl"],
```

Comment:
will the contentUrl be there everytime?
if not, let's add a fallback to empty strings

Reply:
Yes it will be present everytime

Reply:
let's use get anyways..
```suggestion
                content_url=chart.get("contentUrl", ""),
```